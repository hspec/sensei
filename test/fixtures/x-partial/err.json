{"version":"1.0","ghcVersion":"ghc-9.10.0","span":{"file":"test/fixtures/x-partial/Foo.hs","start":{"line":2,"column":7},"end":{"line":2,"column":11}},"severity":"Warning","code":63394,"message":["In the use of `head'\n(imported from Prelude, but defined in GHC.Internal.List):\n\"This is a partial function, it throws an error on empty lists. Use pattern matching, 'Data.List.uncons' or 'Data.Maybe.listToMaybe' instead. Consider refactoring to use \"Data.List.NonEmpty\".\""],"hints":[]}
