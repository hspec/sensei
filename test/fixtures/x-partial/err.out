test/fixtures/x-partial/Foo.hs:2:7: warning: [GHC-63394] [-Wx-partial]
    In the use of `head'
    (imported from Prelude, but defined in GHC.Internal.List):
    "This is a partial function, it throws an error on empty lists. Use pattern matching, 'Data.List.uncons' or 'Data.Maybe.listToMaybe' instead. Consider refactoring to use "Data.List.NonEmpty"."

